container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -148,9 +148,13 @@ releases:
   url: "https://bosh.io/d/github.com/cloudfoundry-incubator/bits-service-release?v=2.28.0"
   sha1: "bd173fc9952277ea962a330c04555d9bd1e8a10e"
 - name: "eirini"
-  url: "https://suse-final-releases.s3.amazonaws.com/eirini-release-0.0.17.tgz"
+  url: "https://s3.eu-central-1.amazonaws.com/tmp-aljfkdjadk-eirinifs-bucket/eirini-bosh-release-dev-b21b0c6.tgz"
```

Comment:
This is just for testing and will be replaced with a final-release before this PR is merged, right?

Reply:
yes, letting Jenkins build a chart to try out on a local cluster. Will reolace with a final release if it works

Reply:
It didn't work with the cri-o kind version, worked on the containerd kind version and also worked on CaaSP 4 (crio). The opi registry certificate was not working for some reason on crio-kind but this might be a problem with kind since it works on CaaSP. We will investigate more but this PR should be ok. It changes nothing relevant to the opi registry anyway.